bears/coffee_script/CoffeeLintBear.py

```diff
@@ -22,11 +22,280 @@ class CoffeeLintBear:
     CAN_DETECT = {'Syntax', 'Formatting', 'Smell'}
 
     severity_map = {'warn': RESULT_SEVERITY.NORMAL,
-                    'error': RESULT_SEVERITY.MAJOR}
+                    'error': RESULT_SEVERITY.MAJOR,
+                    'ignore': RESULT_SEVERITY.INFO}
 
     @staticmethod
     def create_arguments(filename, file, config_file):
-        return '--reporter=raw', filename
+        return '--reporter=raw', '-f', config_file, filename
+
+    @staticmethod
+    def generate_config(filename, file,
+                        max_line_length: int=79,
+                        max_line_length_affect_comments: bool=True,
+                        space_before_and_after_arrow: bool=True,
+                        check_braces_spacing: bool=False,
+                        braces_spacing_width: int=1,
+                        spacing_in_empty_braces: int=1,
+                        class_naming_camelCase: bool=True,
+                        spaces_before_and_after_colon: bool=False,
+                        spaces_before_colon: int=0,
+                        spaces_after_colon: int=0,
+                        enforce_newline_at_EOF: bool=False,
```

Comment:
i think this should be True? else gitmate will throw errors to prove why you need one ;d 
http://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline for more info